src/main/java/org/stellar/sdk/Transaction.java

```diff
@@ -357,6 +368,15 @@ public Builder setTimeout(long timeout) {
       return this;
     }
 
+    public Builder setBaseFee(int baseFee) {
+      if (baseFee < MINIMUM_BASE_FEE) {
```

Comment:
And  then I think `MINIMUM_BASE_FEE` should probably just be `BASE_FEE`. Also ideally it could be fetched from the last ledger like in the [js change](https://github.com/stellar/js-stellar-sdk/pull/234).

Reply:
moved this issue to https://github.com/stellar/java-stellar-sdk/issues/181 as discussed.